selfdrive/car/honda/values.py

```diff
@@ -598,6 +602,7 @@ class CAR:
       b'46114-TLA-A050\x00\x00',
       b'46114-TLA-A930\x00\x00',
       b'46114-TMC-U020\x00\x00',
+      b'39494-TGL-G030\x00\x00',
```

Comment:
Revert this change. On some cars, we don't include all the possible fw queries, and you put this one is in the Civic_diesel

Reply:
Removed `39494-TGL-G030\x00\x00` from the `electricBrakeBooster`.